src/microscope_tiling_basler_camera/src/microscope_tiling_basler_camera/path_builder.py

```diff
@@ -0,0 +1,114 @@
+"""PathBuilder module"""
+
+import numpy as np
+from numpy.typing import NDArray
+from zaber_motion import Units
+from example_util import (
+    convert_length_to_microns,
+    convert_point_to_microns,
+)
+
+
+class PathBuilder:
+    """
+    PathBuilder provides functionality for generating paths for
+    camera tiling
+    """
+
+    Path = list[list[tuple[float, float]]]
+
+    def __init__(
+        self,
+        pxl_w_um: float,
```

Comment:
maybe call this `pixel` intead of shortening it to `pxl`?  You are not saving many characters and imposing a cognitive load on the reader of the code.